test/src/sdk-harness/test_projects/storage_bytes/mod.rs

```diff
@@ -0,0 +1,190 @@
+use fuels::prelude::*;
+
+abigen!(Contract(
+    name = "TestStorageBytesContract",
+    abi = "test_projects/storage_bytes/out/debug/storage_bytes-abi.json",
+));
+
+async fn test_storage_bytes_instance() -> TestStorageBytesContract {
+    let wallet = launch_provider_and_get_wallet().await;
+    let id = Contract::deploy(
+        "test_projects/storage_bytes/out/debug/storage_bytes.bin",
+        &wallet,
+        TxParameters::default(),
+        StorageConfiguration::with_storage_path(Some(
+            "test_projects/storage_bytes/out/debug/storage_bytes-storage_slots.json".to_string(),
+        )),
+    )
+    .await
+    .unwrap();
+
+    TestStorageBytesContract::new(id.clone(), wallet)
+}
+
+#[tokio::test]
+async fn returns_empty_bytes() {
+    let instance = test_storage_bytes_instance().await;
+
+    let input = vec![];
+
+    assert_eq!(instance.methods().len().call().await.unwrap().value, 0);
+
+    let _ = instance
+        .methods()
+        .assert_stored_bytes(input.clone())
+        .call()
+        .await;
+}
+
+#[tokio::test]
+async fn stores_byte() {
+    let instance = test_storage_bytes_instance().await;
+
+    let input = vec![1u8];
+
+    assert_eq!(instance.methods().len().call().await.unwrap().value, 0);
+
+    let _ = instance.methods().store_bytes(input.clone()).call().await;
+
+    assert_eq!(instance.methods().len().call().await.unwrap().value, 1);
```

Comment:
I would consider getting the length of the vector instead of hardcoding a value, this future proofs the code when more values may be added later (unlikely)